app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,8 @@
+class MoviesController < ApplicationController
+  def show
+    @movie = Movie.find(params[:id])
+    @actors = @movie.actors.order(age: :asc)
+    @average_age = @actors.average(:age)
```

Comment:
The ActiveRecord here indicates we should create a method in our model for ordering a movie's actors by age and finding their average age. The AR that you've written is correct, it's just not in the right place.